model/types.go

```diff
@@ -65,34 +66,34 @@ const memWorking = "memory-working"
 const fsLimit = "fs-limit"
 const fsUsage = "fs-usage"
 
-// Request Types.
-// Used as parameters to all the Get methods of the model.
-type ClusterRequest struct {
+// epsilon values for the underlying in-memory stores
+const cpuLimitEpsilon = 10
+const cpuUsageEpsilon = 10
+const memLimitEpsilon = 4194304   // 4 MB
+const memUsageEpsilon = 4194304   // 4 MB
+const memWorkingEpsilon = 4194304 // 4 MB
+const fsLimitEpsilon = 10485760   // 10 MB
+const fsUsageEpsilon = 10485760   // 10 MB
+const defaultEpsilon = 100        // used for testing
```

Comment:
As of now, filesystem metrics default to defaultEpsilon. I will address this in an upcoming PR